tests/test_cli_compile.py

```diff
@@ -1792,6 +1792,32 @@ def test_forwarded_args(PyPIRepository, runner):
     assert set(pip_args).issubset(set(args[0]))
 
 
+@pytest.mark.parametrize(
+    ("pip_args"),
```

Comment:
`("str")` is effectively just `"str"`
```suggestion
    "pip_args",
```